packages/nimble-components/src/table/components/group-row/index.ts

```diff
@@ -29,6 +27,9 @@ export class TableGroupRow extends FoundationElement {
     @observable
     public nestingLevel = 0;
 
+    @observable
+    public dataIndex?: number;
```

Comment:
This feels like a misnomer. A group row has no index in the data.

Reply:
Renamed.